js/gitControl.js

```diff
@@ -6,12 +6,14 @@ gitControl.repos = [];
 
 gitControl.getRepo = function(callback) {
   $.ajax({
-    url: 'https://api.github.com/users/geoffreycc/repos',
+    url: 'https://api.github.com/user/repos', //  Change url to match the one mentioned in class.  user/repos + api key
     type: 'GET',
-    header: {'Authorization': 'token ' + gitHubKey},
+    header: {'Authorization': 'token ' + process.env.gitHubKey}, // Remove this header? Call on something else here? process.env.GITHUBKEY
```

Comment:
ideally this header would be included in your server.js file, not in this call.

look at class code for the class 13 assignment for example